pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -240,16 +240,19 @@ func (m *ManagerImpl) Devices() map[string][]pluginapi.Device {
 // from the registered device plugins.
 func (m *ManagerImpl) Allocate(node *schedulercache.NodeInfo, attrs *lifecycle.PodAdmitAttributes) error {
 	pod := attrs.Pod
+	devicesToReuse := make(map[string]sets.String)
```

Comment:
nit: can we add new field `devicesToReuse` in `m.podDevices` or `ManagerImpl` , so we can stop passing `devicesToReuse` everywhere :)